src/FederationSetup/FederationSetup.cs

```diff
@@ -31,21 +31,29 @@ public static void OutputMenu()
+            builder.AppendLine("              passphrase: a passphrase used to derive the private key from the transaction signing mnenmonic");
+            builder.AppendLine("              datadir:    optional arg, directory where private key is saved");
+            builder.AppendLine("              ismultisig: optional arg, controls output");
+            builder.AppendLine("              Example:    p -passphrase=h@rd2Gu3ss!");
+            builder.AppendLine("              Example:    p -passphrase=h@rd2Gu3ss! -datadir=c:\\dev -ismultisig=true");
             builder.AppendLine("m       Create multi signature addresses for the federation wallets.");
             builder.AppendLine("        args: [-network=<network>] [-quorum=<quorum>] [-fedpubkeys=<pubkey1, pubkey2, ..>]");
             builder.AppendLine("              network:    mainnet, testnet or regtest.");
             builder.AppendLine("              quorum:     The minimum number of federated members needed to sign transactions.");
-            builder.AppendLine("              fedpubkeys: Federation members' public keys. Must have an odd number of up to fifteen members."); // // fed admin will do -m and number (3 qurom + the public keys for the signing of transactions)
+            builder.AppendLine("              fedpubkeys: Federation members' public keys. Up to fifteen members."); // // fed admin will do -m and number (3 qurom + the public keys for the signing of transactions)
             builder.AppendLine("              Example:    m -network=testnet -quorum=2 -fedpubkeys=PublicKey1,PublicKey2,PublicKey3,PublicKey4,PublicKey5");
+            builder.AppendLine("r       Creates a transaction to move the multisig funds to a new federation. The transaction is stored in a (.hex) file in the data directory together with similar files obtained from other nodes.");
+            builder.AppendLine("        args: [-network=<network>] [-datadir=<datadir>] [-quorum=<quorum>] [-fedpubkeys=<pubkey1, pubkey2, ..>] [-password=<password>] [-txtime=<txtime>]");
+            builder.AppendLine("              network:    mainnet, testnet or regtest.");
+            builder.AppendLine("              datadir:    optional arg, old federation directory where private key is saved.");
+            builder.AppendLine("              quorum:     The minimum number of new federation members needed to sign transactions.");
+            builder.AppendLine("              fedpubkeys: New federation members' public keys. Up to fifteen members."); // // fed admin will do -m and number (3 qurom + the public keys for the signing of transactions)
+            builder.AppendLine("              password:   The old federation wallet password.");
+            builder.AppendLine("              txtime:     The timestamp to add to the generated transaction. Corresponds to the agreed time when the old federation will be manually stopped.");
```

Comment:
Hmm why do we need this? Why not just use the time when the tool is run?

Reply:
This ensures the transaction is generated with the same transaction time by everyone. It also acts as a planned cut-off time for switch-over in the sense that we don't include any deposits made after that time.